modules/axios/plugin.js

```diff
@@ -118,6 +136,23 @@ export default (ctx) => {
   });
   <% } %>
 
+  <% if(options.debug) { %>
+  axios.interceptors.request.use(config => {
+    debug('[@nuxtjs/axios] Request:', config)
```

Comment:
Also, I can imagine a general logging helper will be added? Similar like the `warn` [helper](https://github.com/bootstrap-vue/bootstrap-vue/blob/master/lib/utils/warn.js) in `bootstrap-vue`, maybe extended with `chalk` to prit nice colours as well?